scripts/jenkins-yoctobuild-build.sh

```diff
@@ -82,6 +82,12 @@ EOF
 # Build Go repositories.
 export GOPATH="$WORKSPACE/go"
 for build in deployments deviceadm deviceauth inventory useradm; do (
+
+    # If we are testing a specific microservice, only build that one.
+    if [[ ! -z $MICROSERVICE_TO_TEST && $build != $MICROSERVICE_TO_TEST ]]; then
+        continue
+    fi
+
```

Comment:
Intentional, yes, the building and containerizing of the microservices takes a lot of time compared to those two